src/js/providers/html5.js

```diff
@@ -520,6 +557,7 @@ define([
             _source = _levels[_currentQuality];
             _position = item.starttime || 0;
             _duration = item.duration || 0;
+            _itemTracks = item.tracks;
```

Comment:
Look we're passing item `_setVideotagSource`. Why not use it? We should stop adding additional vars to this module's scope and if we really need to just store `_item`. There is no reason to make this a variable in this scope. You can do everything by passing references - functional and can't be tampered with outside of each functions' scope.